jupyterhub/values.yaml

```diff
@@ -82,6 +84,12 @@ rbac:
 
 proxy:
   secretToken: ''
+  deploymentStrategy:
+    ## type: Recreate
+    ## - jupyterhub's interaction with the CHP proxy requires this, if there
+    ##   would be multiple recplicas of the CHP proxy the hub would update and
```

Comment:
Is replica the right word here? It makes me think of having several replicas of the same version of the pod. For example one deployment can have multiple replicas. Here I think we are talking about having one pod per version but switching from communicating with the v1 pod to communicating with the v2 pod "at a bad moment in time".

Maybe "instances" is better? Not super sure what the right k8s lingo is though :(